.github/workflows/main.yaml

```diff
@@ -53,10 +53,6 @@ jobs:
         make tools
         echo "${PWD}/hack/tools/bin" >> $GITHUB_PATH
 
-    - name: Test
-      run: |
-        make all
```

Comment:
`make all` also runs `make cross-build` but I don't see that step in the new workflow.

Reply:
update the workflow with `Cross build`